modules/docs/arrow-docs/docs/docs/datatypes/either/README.md

```diff
@@ -51,6 +51,7 @@ right
 ```kotlin:ank
 val left: Either<String, Int> = Either.Left("Something went wrong")
 left
+}
```

Comment:
Why does it need this unmatched bracket?

Reply:
Woops!  That was a leftover of some edits to the snippet, fixed on 87f9f6a